src/django/api/migrations/0225_add_polygons_switch.py

```
@@ -0,0 +1,31 @@
+    Switch = apps.get_model('waffle', 'Switch')
+    Switch.objects.filter(name=SWITCH_NAME).delete()
+
+
+class Migration(migrations.Migration):
+    """
+    Migration to introduce a switch gating the polygon admin.
+    """
+
+    dependencies = [
+        ('api', '0224_create_polygon'),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)